admin-manual/maintenance/cli-tools.rst

```diff
@@ -2728,6 +2728,73 @@ terminal.
 
 :ref:`Back to top <maintenance-cli-tools>`
 
+.. _cli-tools-data-integrity:
+
+Find data integrity issues
+==========================
+
+AtoM can occasionally end up with missing or incorrect information in the
+database when large operations fail or are aborted mid-process - for example,
+attempting a large :ref:`move <move-archival-description>`,
+:ref:`publication status update <publish-archival-description>`,
+or :ref:`import <import-export>` via the web-based :term:`user interface`, and
+having it time out before the task completes.
+
+This task tries to find data integrity issues in the AtoM database, and
+attempts to repair them if requested.
+
+.. code:: bash
+
+   php symfony tools:data-integrity-repair /path/to/report/filename.csv
+
+
+The data integrity and repair task does the following:
+
+* Adds missing object rows for all resources extending QubitObject
+* Regenerates slugs to use them in CSV report
+* Adds missing parent ids to terms
+* Checks descriptions with missing data and provides options for attempting to
+  generate a list, fix them, or delete them
+* Re-builds the nested sets
+
+If the tool finds any records that have problems, they will be listed in a CSV
+report file. By default, the task saves them in a file named
+``affected-records.csv`` but a custom filename and path can be specified if the
+user wants to save this in a different location.
+
+The ``--application``, ``--env``, and ``connection`` options **should not be
+used** - AtoM requires the use of the pre-set defaults for Symfony to be
+able to execute the task.
+
+The data integrity repair tool has 3 modes. By default it only generates
+a report listing records that have problems and doesn't make any changes to
+them. It is recommended to this first and look through the report before using
```

Comment:
Missed a word in there - It is recommended to [DO] this first 

or: try, attempt, run, etc.